db/migrate/20150718155400_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
```

Comment:
we should not allow nil values for any fields :

:user_id, null: false
:link_id, null: false
:content, null: false